set.mm

```diff
@@ -74424,23 +74431,28 @@ also define a subset of the complex numbers ( ~ df-nn ) with analogous
 
   ${
     $d x y z $.
-    $( Omega is ordinal.  Theorem 7.32 of [TakeutiZaring] p. 43.  (Contributed
-       by NM, 18-Oct-1995.)  (Proof shortened by Andrew Salmon,
+    $( The class of finite ordinals ` _om ` is a transitive class.
+       (Contributed by NM, 18-Oct-1995.)  (Proof shortened by Andrew Salmon,
```

Comment:
Why was "Theorem 7.32 of [TakeutiZaring] p. 43." removed?

Reply:
Sorry, this is in the comment of the next theorem - the diff view was somehow confusing...